### DISCUSSION TITLE
Any plans to support Prism (.NET MAUI) in Visual Code

### DISCUSSION THREAD

**Discussion by quanvm97** [PrismLibrary/Prism#2961] [.NET MAUI]
As latest announcement from MS, Visual Studio for Mac (which is IDE to develop Xamarin framework on Mac) will be retired on Aug 31st, 2024, have to use VS Code with C# Dev Kit and .NET MAUI extensions instead, and only support MAUI/ dotNET framework.

--> Are there any plans to support to create a MAUI Prism project in VS Code for mac?

[Visual Studio for mac Retirement Announcement](https://devblogs.microsoft.com/visualstudio/visual-studio-for-mac-retirement-announcement/)

**Comment by brianlagunas** [comment]
Prism is just a framework and doesn't care what IDE you use.

**Reply by dansiegel** [reply]
@quanvm97 remember at the end of the day it is just another .NET MAUI app so all of the tooling Microsoft has to build a .NET MAUI app continues to work even when you're using Prism, or the Community Toolkit, or Mopups, or Shiny, etc...

**Reply by quanvm97** [reply]
@dansiegel Yes, I got. So I will create a .NET MAUI app, then edit the csproject to add "prism" package and replace MauiAppBuilder with PrismAppBuilder instead of we can create directly a "prism .net maui app" before, right? 
Even if I want to add a new Prism Module in VS Code, I have to setup manually?
